### PR TITLE
Update minikube.md to cover use behind a web proxy

### PR DESCRIPTION
**Pull request by gavinB-orange**
In a corporate environment you need to pass proxy information, and just setting the appropriate environment variables does not work.
The code snippet here works for me

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1370)

### CODE REVIEW

**Comment by dlorenc** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/1370)*

**Comment by dlorenc** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/1370#-:-KTFj5ePFqPTxf1neS3-:bnfp4nl)*

File: docs/getting-started-guides/minikube.md [hunk 1]
```diff
@@ -99,6 +99,14 @@ Kubectl is now configured to use the cluster.
 
 This will build and start a lightweight local cluster, consisting of a master, etcd, Docker and a single node.
 
+If you are behind a web proxy, you will need to pass this information in e.g. via
```
Comment (jaredbhatti) [inline docs/getting-started-guides/minikube.md]: "in e.g. via" is a strange phrasing. 

Perhaps removing this phrase and just leaving it as "you need to pass this information to minikube, following the example below:"

**Comment by jaredbhatti** [comment]
Open doc issue (pretty small) and please accept the CLA.

**Comment by devin-donnelly** [comment]
@gavinB-orange , please fix the merge conflict and the open docs comment, and I can merge.

**Comment by gavinB-orange** [comment]
re. CLA  I am covered by my employer Hewlett Packard Enterprise

**Comment by gavinB-orange** [comment]
Having problems resolving the conflict - pressing the "Resolve conflicts" button just leads to a spinning GitHub wait icon ...

**Comment by jaredbhatti** [comment]
Huh, can you try again?  Pressing "resolve conflicts" seems to work for me.

**Pull request merged**